eng/pipelines/repo-mirror.yml

```diff
@@ -0,0 +1,121 @@
+trigger: none
+pr: none
+
+# Sync every 2 hours
+schedules:
+  - cron: "0 */2 * * *"
+    branches:
+      include:
+      - master
+    always: true
+
+jobs:
+  - job: MirrorRepo
+    pool:
+      vmImage: 'Ubuntu 18.04'
+    variables:
+      skipComponentGovernanceDetection: true
+
+    steps:
+      - checkout: none
+
+      - template: ./templates/steps/repo-mirror.yml
+        parameters:
+          WORKING_DIRECTORY: $(WORKING_DIRECTORY)
+          SOURCE_REPO: $(SOURCE_REPO)
+          SOURCE_BRANCHES: $(SOURCE_BRANCHES)
+          TARGET_ENDPOINTS: $(TARGET_ENDPOINTS)
+
+    strategy:
+      matrix:
+
+        azure_rest_api_specs:
+          WORKING_DIRECTORY: repos/azure-rest-api-specs
+          SOURCE_REPO: https://github.com/Azure/azure-rest-api-specs
+          SOURCE_BRANCHES: master
+          TARGET_ENDPOINTS: |
+            https://github.com/Azure/azure-rest-api-specs-pr!master
+            https://github.com/test-repo-billy/azure-rest-api-specs-pr!master
+            https://github.com/test-repo-billy/azure-rest-api-specs!master
+            https://github.com/openapi-env-test/azure-rest-api-specs!master
+
+        azure_sdk_for_go:
+          WORKING_DIRECTORY: repos/azure-sdk-for-go
+          SOURCE_REPO: https://github.com/Azure/azure-sdk-for-go
+          SOURCE_BRANCHES: master latest
+          TARGET_ENDPOINTS: |
+            https://github.com/Azure/azure-sdk-for-go-pr!master
+            https://github.com/AzureSDKAutomation/azure-sdk-for-go!master
+            https://github.com/test-repo-billy/azure-sdk-for-go!master
+            https://github.com/openapi-env-test/azure-sdk-for-go!master
+            https://github.com/Azure/azure-sdk-for-go-pr!latest
+            https://github.com/AzureSDKAutomation/azure-sdk-for-go!latest
+            https://github.com/test-repo-billy/azure-sdk-for-go!latest
+            https://github.com/openapi-env-test/azure-sdk-for-go!latest
+
+        azure_sdk_for_js:
+          WORKING_DIRECTORY: repos/azure-sdk-for-js
+          SOURCE_REPO: https://github.com/azure/azure-sdk-for-js
+          SOURCE_BRANCHES: master
+          TARGET_ENDPOINTS: |
+            https://github.com/Azure/azure-sdk-for-js-pr!master
+            https://github.com/azuresdkautomation/azure-sdk-for-js!master
```

Comment:
Why do we have all these forks? Are they necessary? Why can't we just use the forks from our bot accounts instead of having so many different forks.

Reply:
AzureSDKAutomation is for holding automation branches. It is a githubApp so it don't have a org. test-repo-billy and openapi-env-test are our test org that we do need them to keep synced with upstream. We plan to merge them into one test org but not done yet.

Reply:
We should try to consolidate on using the forks of one our bots such as https://github.com/azure-sdk/azure-sdk-for-net instead of having these forks spread all over the place were we have to keep them all in sync and manage them.